### PR TITLE
Updating participant information for 2023 October

### PR DESCRIPTION
**Pull request by Jen-OMalley**
Updated participant information for this workshop and made a few small tweaks elsewhere. This PR addresses [this issue](https://github.com/AlexsLemonade/training-admin/issues/1016), that was filed in `training-admin`.

### CODE REVIEW

File: docs/participant_information.md
Comment (sjspielman) [inline docs/participant_information.md:21]: Maybe, "Tasks to complete before the workshop"? I'm not sure "action items" conveys this as clearly?

File: docs/participant_information.md
Comment (sjspielman) [inline docs/participant_information.md:35]: please consider this comment my vote for [Manakeesh](https://www.manakeeshcafe.com/) 🤤

File: docs/participant_information.md
Comment (sjspielman) [inline docs/participant_information.md:67]: Hm, the latter is a vaccination rate link. How about this one?

```suggestion
You may wish to review the [CDC COVID-19 by County guidance](https://www.cdc.gov/coronavirus/2019-ncov/your-health/covid-by-county.html) and the [Philadelphia, Pennsylvania COVID-19 community level](https://covid.cdc.gov/covid-data-tracker/#trends_weeklyhospitaladmissions_select_00) when planning for workshop attendance.
```

File: docs/setup_instructions/setup_overview.md
Comment (sjspielman) [inline docs/setup_instructions/setup_overview.md:21]:
```suggestion
If you need any assistance with the installation, do not hesitate to reach out to the Data Lab team by email, or, even better, in the [Cancer Data Science Slack](http://ccdatalab.org/slack). We will add you to a training-specific channel after you join the Slack community.
```

**Review by sjspielman — COMMENTED** [review]
This all looks fine to me! Left a couple text suggestions and one comment just for fun 😄 
I'm just holding off on approving because I want to check back with the issue this is addressing (see my other comment - link isn't working?), but I don't anticipate really having much else to say!

File: docs/participant_information.md
Comment (sjspielman) [reply docs/participant_information.md:35]: Wait they do catering?! https://order.toasttab.com/online/manakeesh-cafe-bakery-grill#dfca412c4-d782-4a16-9562-1531165a4f18d63ce381c-e43e-4ff9-8b26-85d64733b21b
please therefore consider this my vote to get them to cater something at some point in the future ;)

**Comment by Jen-OMalley** [comment]
> @Jen-OMalley the link to the `training-admin` issue doesn't appear to be working - can you comment with the right link? Thanks!

Oops! It should be this issue. https://github.com/AlexsLemonade/training-admin/issues/1016

**Review by sjspielman — APPROVED** [review]
Looks good for the issue, so approving now assuming you'll catch my reviews! lmk if anything is unclear or you disagree with something!

**Pull request merged**
